src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -1450,6 +1475,33 @@ public String getNotificationLevel() {
     }
 
     /**
+     * Silent Mode level to use.
+     *
+     * @return the silent level value
+     */
+    public SilentLevel getSilentLevel() {
+        return silentLevel;
+    }
+
+    /**
+     * Should comments be kept silent
+     *
+     * @return whether comments are kept silent
+     */
+    public boolean isSilentComments() {
+        return (silentMode && (silentLevel == SilentLevel.ALL || silentLevel == SilentLevel.COMMENTS));
```

Comment:
Instead of adding even more code to this class I think this logic is better put in static utility methods in the enum class.